assets/js/more-link.js

```diff
@@ -0,0 +1,8 @@
+( function() {
+	var links = document.querySelectorAll('.more');
```

Comment:
Can we follow the WP coding standards here?

```suggestion
	var links = document.querySelectorAll( '.more' );
```